.github/ISSUE_TEMPLATE/special_case_issue.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Special Case Issue
+about: An complex or exceptional issue that doesn't fit the standard templates
+
+---
+
+<!-- 
+Please read the text in this edit field before filling it in. 
+If your issue doesn't neatly fit into either the bug report or feature request categories, and it involves a more complex or unique scenario, use this template. Before proceeding, please review the existing templates to ensure that those do not better suit your needs:
+- Bug Report Template: https://github.com/nvaccess/nvda/blob/master/.github/ISSUE_TEMPLATE/bug_report.md
```

Comment:
Since the tone of this template STRONGLY encourages people to use one of the other ones, might it be better to link to the actual fillable templates here, instead of the files themselves? It is unlikely someone will want to edit the template.

```suggestion
- Bug Report Template: https://github.com/nvaccess/nvda/issues/new?assignees=&labels=&projects=&template=bug_report.md
```